simplified-app-simplye/src/main/assets/Accounts.json

```diff
@@ -58,5 +58,23 @@
     "catalogUrl13": "https://instantclassics.librarysimplified.org/all-books.xml",
     "licenseUrl": "http://www.librarysimplified.org/iclicenses.html",
     "mainColor": "#497049"
+  },
+  {
+    "id": 4,
```

Comment:
Library ID 4 is already taken by the Canadian Electronic Library, in a branch that has not been merged yet.

To avoid future confusion of this type I've established a list of library IDs here: https://github.com/NYPL-Simplified/Simplified/wiki/UniqueLibraryIDs

I've assigned the ID `8` to Howard County library.